src/oci-umount.c

```diff
@@ -580,6 +580,18 @@ static int parseBundle(yajl_val *node_ptr, char **rootfs, struct config_mount_in
 	}
 	*rootfs=strdup(YAJL_GET_STRING(v_root));
 
+	/* Prepend bundle path if the rootfs string is relative */
+	if (*rootfs[0] != '/') {
+		char *new_rootfs;
+
+		asprintf(&new_rootfs, "%s/%s", YAJL_GET_STRING(v_bundle_path), *rootfs);
+		if (!new_rootfs) {
+			pr_perror("failed to alloc rootfs");
+			return EXIT_FAILURE;
+		}
+		*rootfs = new_rootfs;
```

Comment:
👍 to the no '*' thought.  It works without.  Compile time error in my testing, not sure why that wasn't noted by the build process....